src/main/java/org/jfrog/hudson/generic/ArtifactoryGenericConfigurator.java

```diff
@@ -277,6 +309,7 @@ public ArtifactoryServer getArtifactoryResolverServer() {
     public Environment setUp(final AbstractBuild build, Launcher launcher, BuildListener listener)
             throws IOException, InterruptedException {
         listener.getLogger().println("Jenkins Artifactory Plugin version: " + ActionableHelper.getArtifactoryPluginVersion());
+        PublisherContext.Builder publisherBuilder = getBuilder();
```

Comment:
This line seems to be redundant.

Reply:
Yes, you're right. I wasn't sure really with this one as it figures in ArtifactoryGradleConfigurator.java which I used as a sort of model example from where I was taking the existing implementation.